### ISSUE TITLE
Bing imagery is loaded via HTTP instead of HTTPs

### ISSUE THREAD

**Issue by AkeluX** [NASAWorldWind/WebWorldWind#129]
From the forum:

> We transitioned our site from HTTP to HTTPS in January, following the same move by WorldWind. We've been seeing the following error where dependent images are coming through HTTP instead of HTTPS:
> Mixed Content: The page at 'https://REDACTED.com/REDACTED/REDACTED' was loaded over HTTPS, but requested an insecure image 'http://ecn.t3.tiles.virtualearth.net/tiles/a0320102211313.jpeg?g=5716'. This content should also be served over HTTPS.

**Comment by AkeluX** [comment]
We need to explicitly request an HTTPS address for the imagery when querying the service metadata, see https://msdn.microsoft.com/en-us/library/ff701716.aspx. This would look like in f11c9e65aece6a365741d99c5f7097887f57bb16.

Do we know of any issue in requesting HTTPs all the time? Otherwise, we could choose depending on the value of `location.protocol` for example. What do you think?

**Comment by pdavidc** [comment]
Yann, Thanks for looking into this and for identifying the necessary changes. I don't know of any issue requesting HTTPS all the time. My intent has been to transitioning all built-in World Wind layers to use HTTPS all the time, since that is the only protocol compatible with all deployment scenarios.

When I transitioned the Bing layer to HTTPS I overlooked this aspect of Bing imagery metadata. I think the changes in branch fix/bing_https are great as-is.
